release/README.md

```diff
@@ -69,28 +69,30 @@ $ alias githubctl="${PWD}/bazel-bin/toolbox/githubctl/githubctl"
 
 The release process goes like the following:
 
-Step 1: Tag the release.
+### Step 1: Tag the release.
 ```
 $ githubctl --token_file=<github token file> \
     --op=tagIstioDepsForRelease \
     --base_branch=<release branch or master>
 ```
 
-Step 2: The previous command triggers rebuild and retagging on pilot, proxy, mixer and auth.
+### Step 2: The previous command triggers rebuild and retagging on pilot, proxy, mixer and auth.
```

Comment:
Please replace auth with security.

Reply:
We need a .2 version and master version of these instructions checked into the respective branches. That will make this cleaner.

Reply:
@rkpagadala  The release from master which is 0.3 (or maybe 0.4 if they decide to skip 0.3) will happen in the new world and hopefully will be handle by fully-automated release train.

Reply:
@wattli I am not sure if we need to replace this here, since it's for release 0.2, which still happens in old_auth_repo. I am just going to put it in addition.